addons/delivery/models/delivery_price_rule.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields, api
+import openerp.addons.decimal_precision as dp
+
+class PriceRule(models.Model):
+    _name = "delivery.price.rule"
+    _description = "Delivery Price Rules"
+    _order = 'sequence, list_price'
+
+    @api.depends('variable', 'operator', 'max_value')
+    def _get_name(self):
+        for rule in self:
+            name = 'if %s %s %s' % (rule.variable, rule.operator, rule.max_value)
+            rule.name = name
+
+    name = fields.Char(compute='_get_name')
```

Comment:
can not override name_get method instead of make compute method. use display_name field instead of name to display name